src/components/ui/Checkbox/checkbox.scss

```diff
@@ -2,6 +2,7 @@
   display: grid;
   grid-template-columns: auto minmax(0, 1fr);
   align-items: center;
+  gap: var(--spacing-xs);
```

Comment:
Checkbox produces unnecessary spacing if children is not provided

![image](https://github.com/user-attachments/assets/7b9d784e-b70c-451a-a9ef-6ccd9d188951)